### DISCUSSION TITLE
Feature Request:  Support Secret sharing

### DISCUSSION THREAD

**Discussion by yujinqiu** [FiloSottile/age#369] [General]
Hi, 

age is a great cli tool,  really love it <3.  Can support [key sharing](https://en.wikipedia.org/wiki/Secret_sharing) , like this project 
[https://github.com/onetwopunch/sham], so I don't need two cli. 

Why I need this feature ? 
I want to sharing my key to 3 person and once more than 2 people agree to see the content. 

It can be better if age can support  specified  ssh key.

**Comment by covert-encryption** [comment]
Duplicate of #305
